### PR TITLE
Correct the info block handling for the category blog

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue #11882

### Summary of Changes

Correct the info block handling for the category blog

### Testing Instructions

Set Position of Article Info to Below. Go to the website and you'll see the Tags under the title under the title. This happens when you choose Category Blog as Menu Item Type. The Tags are Below when you use the Featured articles as type.

### Expected result

The Tags should be Below the article in Category Blog.

### Actual result

The are Above.

### Documentation Changes Required

none.

### CODE REVIEW

**Comment by rjcf18** [comment]
I have tested this item :white_check_mark: successfully on f3d0568807c27992765707d2ca8153cfd3a91e4f<br /><br />The tags appear below the article in category blog after setting position of article info to Below.

deepinscreenshot20170424031505

deepinscreenshot20170424034425
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15497">issues.joomla.org/tracker/joomla-cms/15497</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on f3d0568807c27992765707d2ca8153cfd3a91e4f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15497">issues.joomla.org/tracker/joomla-cms/15497</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Comment by rdeutz** [comment]
@zero-24 please check the conflicts, thanks

**Comment by zero-24** [comment]
Retests are required!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15497">issues.joomla.org/tracker/joomla-cms/15497</a>.</sub>

**Comment by rjcf18** [comment]
I have tested this item :white_check_mark: successfully on 9afd107f51b9ba9c02a8a7306a08374b782ff6fe<br /><br />The tags appear below the article after changing position of article info to below.

## Before:
deepinscreenshot20170522214127

## After:
deepinscreenshot20170522214209
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15497">issues.joomla.org/tracker/joomla-cms/15497</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 9afd107f51b9ba9c02a8a7306a08374b782ff6fe<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15497">issues.joomla.org/tracker/joomla-cms/15497</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

File: components/com_content/views/category/tmpl/blog_item.php [hunk 1]
```diff
@@ -30,33 +30,37 @@
...
 <?php // Content is generated by content plugin event "onContentBeforeDisplay" ?>
 <?php echo $this->item->event->beforeDisplayContent; ?>
 
 <?php echo $this->item->introtext; ?>
 
 <?php if ($useDefList && ($info == 1 || $info == 2)) : ?>
 	<?php // Todo: for Joomla4 joomla.content.info_block.block can be changed to joomla.content.info_block ?>
-	<?php echo JLayoutHelper::render('joomla.content.info_block.block', array('item' => $this->item, 'params' => $params, 'position' => 'below')); ?>
-<?php  endif; ?>
+	<?php echo JLayoutHelper::render('joomla.content.info_block.bloc', array('item' => $this->item, 'params' => $params, 'position' => 'below')); ?>
```
Comment (richard67) [inline components/com_content/views/category/tmpl/blog_item.php]: 'joomla.content.info_block.bloc'? Not 'joomla.content.info_block.bloc**k**'?

Comment (brianteeman) [inline components/com_content/views/category/tmpl/blog_item.php]: looks like a typo

Comment (richard67) [inline components/com_content/views/category/tmpl/blog_item.php]: That's why I commented it. Unfortunately I do not have the time to test that in order to get a negative test result. I hope @zero-24 will notice and correct it anyway.

**Comment by zero-24** [comment]
Thanks!

**Pull request merged**